apps/api/src/utils/buildQueryParams.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Copyright

Reply:
Didn't mean to commit this file. Its not used. Removing.